src/terrain/clients/app_exposer.clj

```diff
@@ -153,4 +153,36 @@
   [id]
   (-> (app-exposer-url ["instantlaunches" id] {} :no-user true)
       (client/delete {:as :json})
+      (:body)))
+
+;;; We bypass the query map handling built into (app-exposer-url) to use the
+;;; one from clj-http because the former does not handle having an seq of
+;;; values for an entry while the latter does.
+(defn list-metadata
```

Comment:
Oh, I guess the one question would be if this should specify that it's for instant launch metadata specifically. Neglected to notice it's not in an IL-specific namespace

Reply:
I added a bit of clarification to the comment.